tests/conftest.py

```diff
@@ -48,3 +49,30 @@ def undo():
             monkeypatch.setattr('click.echo', old_echo)
 
     return Result
+
+
+@pytest.fixture
+def sleep_time(tmpdir):
+    """
+    Returns how long we need to sleep for the filesystem's mtime precision to
+    pick up differences.
+
+    This keeps test fast on systems with high precisions, but makes them pass
+    on those that don't.
+    """
+    tmpfile = tmpdir.join('touch_me')
+
+    def touch_and_mtime():
+        with tmpfile.open('w') as f:
```

Comment:
This and the next line can be replaced by `tmpfile.ensure(file=True)` which is equivalent to `touch`.